scripts/s3/s3api/s3_obj_range_read_io_stability.py

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+"""File contains S3 object range read operations test script for io stability."""
+
+import logging
+import os
+import random
+import time
+from datetime import datetime, timedelta
+from typing import Union
+
+from botocore.exceptions import ClientError
+
+from src.libs.s3api.s3_bucket_ops import S3Bucket
+from src.libs.s3api.s3_object_ops import S3Object
+
+logger = logging.getLogger(__name__)
+
+
+# pylint: disable=too-many-arguments, too-many-locals, too-many-instance-attributes
+# pylint: disable=too-few-public-methods, too-many-statements
+class TestObjectRangeReadOps(S3Object, S3Bucket):
+    """S3 Object range read operations class for executing given io stability workload"""
+
+    # pylint: disable=too-many-arguments, too-many-locals, too-many-instance-attributes
```

Comment:
I see this rules already disabled on line no 37, keep either one but I would recommend to resolve them as much as possible instead skipping them.

Reply:
done